backend/de.metas.fresh/de.metas.fresh.base/src/main/sql/postgresql/system/70-de.metas.fresh/5726940_sys_gh18285_Use_Lookup_Instead_Listview.sql

```diff
@@ -0,0 +1,16 @@
+select backup_table('ad_column'); -- backup.ad_column_bkp_20240620_114638_818
+select backup_table('ad_field'); -- backup.ad_field_bkp_20240620_114658_431
+
+UPDATE AD_Field f
+SET AD_Reference_ID = 30
+WHERE f.AD_Reference_ID IN (18, 19)
+  AND f.AD_Column_ID NOT IN (SELECT AD_Column_ID
+                             FROM AD_Column
+                             WHERE AD_Reference_ID = 30)
```

Comment:
so this UPDATE will update all fields
and will set reference=Search
if 
* the field reference is Table,TableDir (i.e. not Search)
* and the column reference is Search

Is that matching the logic provided in requirements? i.e.
![image](https://github.com/metasfresh/metasfresh/assets/1244701/c225c91c-7a52-4413-9c34-89c73de2f5a7)

Reply:
yes

Reply:
not so sure... am i missing something?

![image](https://github.com/metasfresh/metasfresh/assets/1244701/e4348aa8-1951-4eb6-91af-56025836761a)

Reply:
I understand it like this: `Exclude fields (leave them as list) where fields != search and column=search`:

Reply:
that's right.

now, can u pls check the SQL bellow and pls tell me what's that doing compared to yours:
```sql
UPDATE AD_Field f SET AD_Reference_ID = 30
FROM AD_Column c
WHERE f.AD_Column_ID=c.AD_Column_ID
    AND f.AD_Reference_ID IS NOT NULL
    AND f.AD_Reference_ID=c.AD_Reference_ID
;
```

Reply:
also, before updating you could check and document which are the affected cases.... shall not be so many

Reply:
the list of fields that wil be updated are documented in the dev docu